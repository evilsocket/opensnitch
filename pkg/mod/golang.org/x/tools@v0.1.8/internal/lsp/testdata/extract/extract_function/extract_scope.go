package extract

func _() {
	newFunction := 1
	a := newFunction //@extractfunc("a", "newFunction")
}

func newFunction1() int {
	return 1
}
