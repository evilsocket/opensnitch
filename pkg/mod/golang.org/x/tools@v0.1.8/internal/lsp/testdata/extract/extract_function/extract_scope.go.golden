-- functionextraction_extract_scope_5_2 --
package extract

func _() {
	newFunction := 1
	newFunction2(newFunction) //@extractfunc("a", "newFunction")
}

func newFunction2(newFunction int) {
	a := newFunction
}

func newFunction1() int {
	return 1
}

